tests/FakeItEasy.Tests/Core/CallOrderingContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core

```

Comment:
Namespace (and file location) is a little odd, given we're testing `FakeItEasy.CallOrderingContext`.